src/main/java/com/github/davidcarboni/thetrain/helpers/Configuration.java

```diff
@@ -35,4 +33,11 @@ public static String transactionStore() {
         return StringUtils.defaultIfBlank(System.getenv(TRANSACTION_STORE), get(TRANSACTION_STORE_LEGACY));
     }
 
+    public static int threadPoolSize() {
+        try {
+            return Integer.parseInt(System.getenv(THREAD_POOL_SIZE));
+        } catch (NumberFormatException e) {
```

Comment:
could we log this exception?

also should providing invalid config prevent the app from starting rather than defaulting? e.g. setting it to `100)` instead of `1000` and having it default to `20` could cause nasty problems?

Reply:
done